sktree/tests/test_forest.py

```diff
@@ -41,3 +56,28 @@ def test_urf():
     # XXX: This should be > 0.9 according to the UReRF. Hoewver, that could be because they used
     # the oblique projections by default
     assert score > 0.6
+
+
+def check_iris_criterion(name, criterion):
+    # Check consistency on dataset iris.
+    ForestClassifier = FOREST_CLASSIFIERS[name]
+
+    clf = ForestClassifier(n_estimators=10, criterion=criterion, random_state=1)
+    clf.fit(iris.data, iris.target)
+    score = accuracy_score(clf.predict(iris.data), iris.target)
+    # score = clf.score(iris.data, iris.target)
+    assert score > 0.3, "Failed with criterion %s and score = %f" % (criterion, score)
+
+    clf = ForestClassifier(
+        n_estimators=10, criterion=criterion, max_features=2, random_state=1
+    )
+    clf.fit(iris.data, iris.target)
+    score = accuracy_score(clf.predict(iris.data), iris.target)
```

Comment:
What's the reasoning for using `accuracy_score`? Isn't it a classification measure? 

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

Reply:
Thank you for this! I followed your `adjusted_rand_score method`